### ISSUE TITLE
Test: [DMS] Learned AI model과 dlib model 비교 및 테스트

### ISSUE THREAD

**Issue by jis0202** [66yurimi99/Z-ERO#13]
## Description
학습한 AI Model과 dlib의 모델(frontal_face_detector, 68_facial_landmarks)을 이용한 알고리즘 성능을 비교합니다.
<br>

## Todo
- [x] 비교용 프로그램 작성 및 데이터 생성
- [x] 비교 테스트 후 결과 자료 작성
<br>

## 성능 지표
- Accuracy
- Running time
- Memory 사용량
<br>

## TEST CASE
- CASE 1: Learned face detection model + Learned classification model
- CASE 2: dlib frontal face detector + dlib 68 facial landmarks + EAR

**Comment by jis0202** [comment]
#### Face detector Training 결과
|model name|Device|Accuracy|FPS|Batch size|Learning rate|Epoch|
|----|----|----|----|----|----|----|
|mobilenet-ATSS| CPU | 0.76(70%) / 0.28(80%) / 0.0(90%) | 19 | 8 | 0.004 | 200 |
|mobilenet-ATSS| GPU | 0.76(70%) / 0.32(80%) / 0.0(90%) | 9 | 8 | 0.004 | 200 |
|SSD| CPU | 1.0(90%) / 1.0(99%) | 25 | 8 | 0.01 | 200 |
|SSD| GPU | 1.0(90%) / 1.0(99%) | 10 | 8 | 0.01 | 200 |
|YOLOX| CPU | 1.0(80%) / 0.96(88%) / 0.76(90%) | 44 | 8 | 0.0002 | 200 |
|YOLOX| GPU | 1.0(80%) / 0.96(88%) / 0.72(90%) | 38 | 8 | 0.0002 | 200 |
#### Eyes close classification Training 결과
|model name|Device|Accuracy|FPS|Batch size|Learning rate|Epoch|
|----|----|----|----|----|----|----|
|EfficientNet-B0| GPU | O: 0.92(70%) / 0.92(80%) / 0.89(90%)<br>X: 0.98(70%) / 0.95(80%) / 0.94(90%) | 145 | 64 | 0.0049 | 90 |
|EfficientNet-V2-S| GPU | O: 0.96(70%) / 0.95(80%) / 0.94(90%)<br>X: 0.99(70%) / 0.99(80%) / 0.99(90%) | 183 | 64 | 0.0058 | 90 |
|MobileNet-V3-large-1x| GPU | O: 1.0(77%) / 0.99(80%) / 0.99(90%)<br>X: 1.0(75%) / 0.99(80%) / 0.99(90%) | 55 | 64 | 0.0071 | 90 |

### dlib basic model vs Trained model
|model sets|FPS|O_Accuracy|X_Accuracy|
|----|----|----|----|
|dlib basic models| 9 | 0.32 | 0.16 |
|Trained models| 26 | 1.0 | 1.0 |
